app/controllers/api/automate_workspaces_controller.rb

```diff
@@ -1,8 +1,41 @@
 module Api
   class AutomateWorkspacesController < BaseController
     def edit_resource(type, id, data = {})
+      raise BadRequestError, "Must specify an id for starting a #{type} resource" unless id
```

Comment:
Has this been copy/pasted from elsewhere? Do you mean s/starting/editing/, etc..? Can you add a test for these?

Reply:
@imtayadeway That line is not needed, I had accidentally added it. The resource_search raises the error.